Library/Formula/checkstyle.rb

```diff
@@ -16,7 +16,7 @@ def install
     path.write "public class Foo{ }\n"
 
     output = `#{bin}/checkstyle -c /sun_checks.xml #{path}`
-    errors = output.split("\n").select { |line| line.start_with?(path) }
+    errors = output.split("\n").select { |line| line.start_with?("[ERROR] " + path) }
```

Comment:
Tiny nit: `"[ERROR] #{path}"` is a bit nicer.

Also, not your change but: `output.split("\n")` can be instead `output.lines`